components/BlockContentHandler/styles/BlockContent.module.scss

```diff
@@ -0,0 +1,18 @@
+.content {
+  margin: 50px 0 100px 0;
+  text-align: left;
+}
+
+@media screen and (max-width: 500px) {

```

Comment:
Rekomenduoju susikurt settings.scss faila somwhere ir ten definint visus vars
pvz.: breakpoints, colors
ir tada tiesiog @media screen and (max-width: $breakpoint-tablet)

See https://github.com/In-Real-Life/IRL/tree/master/packages/web/src/assets/scss/settings

Jei kursi tuos failus, juos reikes importuot i kiekviena faila... Bet kad to nereiktu gali automatiskai appendint like this
https://github.com/In-Real-Life/IRL/blob/c3f70a498d07c6ec8937fb2791fda1e8e30fbe91/packages/web/next.config.js#L44


Alsooo.. Naudok em for media queries
http://blog.w3conversions.com/2015/03/all-your-base-are-belong-to-us-%E2%80%94-or-what-base-size-do-em-based-media-queries-use/